src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -987,7 +987,10 @@ Parameter Engine::GetMetric(const std::string& name, const std::map<std::string,
 
         return decltype(ov::supported_properties)::value_type(supportedProperties);
     } else if (name == ov::device::full_name) {
-        return decltype(ov::device::full_name)::value_type(deviceFullName);
+        auto name = decltype(ov::device::full_name)::value_type(deviceFullName);
+        std::string name_str;
+        name_str.append(&(*name.begin()));
+        return name_str;
```

Comment:
why do we need it?

Reply:
After conversations with CPU team this has been reverted and the task for this problem has been created